CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_iterable_source v0.10.0
+[PR #39](https://github.com/fivetran/dbt_iterable_source/pull/39) contains the following updates:
+
+## Bug Fix 
+- Removes the source freshness tests in the `src_iterable.yml`. This was originally causing run errors for users where tables weren't present in their schema but listed in the `src_iterable.yml`.
+- Removes `created_at` from the uniqueness test for `stg_iterable__event`. Uniqueness is tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id`.
+
+## Breaking Change (`--full-refresh` required after upgrading)  
+- Updates the materialization in `stg_iterable__user_history` from table to view in order to improve performance. We encourage you to run a `--full-refresh` to ensure there are no issues.
+
+## Documentation
+- Updates the descriptions of timestamp-based fields. Previously they were described as milliseconds since epoch time, but they should be standard timestamps.
+
+## Contributors
+- [@justin-fundrise](https://github.com/justin-fundrise)
```

Comment:
If there is no PR linked in the source for the contribution then we shouldn't include this section in the CHANGELOG. We should only include a contributing section when there is a PR linked within the package update.

Reply:
Ah ok! That makes a lot of sense